mentoring/public/css/dashboard.css

```diff
@@ -0,0 +1,31 @@
+.pb-dashboard table {
```

Comment:
We'd like XBlocks to get in the habit of prefacing css rules with the block name, to scope down the rules to the block itself - you guys did a great job here: https://github.com/open-craft/xblock-mentoring/blob/master/mentoring/public/css/mentoring.css and would like to see that extend to all CSS.

Also, what does "pb-dashboard" mean? That rule (at least right now) is not clearly named to me.